docs/on-call.md

```diff
@@ -0,0 +1,84 @@
+---
+cover: assets/img/headers/FullService_OnCall.png
+description: A step-by-step guide to prepare full-service ownership teams to go on-call for the first time.
+---
+![Going On-Call](../assets/img/headers/FullService_OnCall.png)
+
+> My greatest fear is getting a high priority page while on-call in a packed broadway theater, and like PagerDuty plays a progressively louder and louder quacking siren, and Alexander Hamilton himself comes up to slap me and my phone out of my hand, and walks me out of the theater” via [Twitter](https://twitter.com/JulianVModesto/status/1222279149819256832)
```

Comment:
maybe we remove the "like" in "and like PagerDuty plays..." I know that the original quote has it, but I think we can remove it for clarity without harm?